arangod/Agency/FailedLeader.cpp

```diff
@@ -115,13 +115,21 @@ void FailedLeader::rollback() {
     // Transactions
     payload = std::make_shared<Builder>();
     {
-      VPackObjectBuilder b(payload.get());
-      for (auto const c : cs) {
-        payload->add(planColPrefix + _database + "/" + c.collection +
-                         "/shards/" + c.shard,
-                     rb.slice());
+      VPackArrayBuilder a(payload.get());
```

Comment:
This does not look right. Earlier, the result was an object, now it is an array...

Reply:
OK, understood, `finish` distinguishes cases.